### DISCUSSION TITLE
Brim question, how do I get a brim to actually connect to the part?

### DISCUSSION THREAD

**Discussion by JeffDM2** [OrcaSlicer/OrcaSlicer#4776] [Q&A] [answered]
![Screenshot 2024-03-29 21 49 24](https://github.com/SoftFever/OrcaSlicer/assets/4670911/d16b77cd-b31f-4500-9644-6bae2247eefd)
I don't understand why the software isn't giving me what I'm asking for, and that's a brim that actually connects to the part. I've also tried negative gap numbers to push it closer but the software rejects that.

I think the other slic3r variants I've used done the same thing.

Normally I don't use brims but this part has such a small contact area I want to tip the odds a bit.

**Comment by Manfredo1952** [comment]
Which version of Orca are you using ? I have the 2.0 RC here and  get this result - i.e. brim attached to the part. 
<img width="1129" alt="grafik" src="https://github.com/SoftFever/OrcaSlicer/assets/56330568/aa1beda8-1380-49c8-a89f-b1518861daa4">

**Reply by JeffDM2** [reply]
I’m using an RC from last week, a couple days after they stopped posting betas. I’ll update and see if it’s any different.

**Reply by JeffDM2** [reply]
I couldn't find a setting or combination of settings that changed it. I even made a plain cube and simple cylinder. Also this is the release version of 2.0, I updated it half an hour ago.

![Screenshot 2024-03-31 03 03 02](https://github.com/SoftFever/OrcaSlicer/assets/4670911/3034ccb7-687a-44b6-b08d-f7429a232670)
![Screenshot 2024-03-31 02 59 10](https://github.com/SoftFever/OrcaSlicer/assets/4670911/83ed8663-5679-415d-8fb0-109f72b202aa)
![Screenshot 2024-03-31 02 55 50](https://github.com/SoftFever/OrcaSlicer/assets/4670911/9b556dfe-704a-4363-acf3-30b7cd2c07d9)

**Comment by Manfredo1952** [comment]
#
You have  set skirt to zero distance. That will probably interfere with the brim
Suggest to set skit to zero loops and then  brim like here  i.e.  5mm  width and 0 distance




￼

**Comment by Manfredo1952** [comment, marked as answer]
ou have  set skirt to zero distance. That will probably interfere with the brim
Suggest to set skit to zero loops and then  brim like here  i.e.  5mm  width and 0 distance




<img width="1101" alt="grafik" src="https://github.com/SoftFever/OrcaSlicer/assets/56330568/e19cbf3d-49d6-4744-acdc-e97deb3f0a4b">
<img width="1101" alt="grafik" src="https://github.com/SoftFever/OrcaSlicer/assets/56330568/92ac9b5d-5738-43fe-9ec4-90f36fdbad4e"

**Reply by JeffDM2** [reply]
OK thanks that fixed it. I guess I never learned the distinction and still don't undersand but I'm going to make this change and worry about understanding it later.

**Reply by Manfredo1952** [reply]
It is actually pretty simple. The skirt is used to print a purge line around the part to enable the nozzle to get rid of any unwanted filament that  got eventually pushed out of the nozzle immediately prior to start of actual printing, mostly due to filament getting hot (printing temp) and thus dripping through the nozzle. The skirt is then printed a few mm off the actual part. The brim is used to make sure that the part does not warp/lift at the egdes due to difference of temperatures.  A heated chamber does help here a lot. Or a brim.    You can also use "mouse ears" (same block of settings), as shown here which adds a brim just at the edges, where this lift of filament might occur.    When you have selected " mouse ear" hit slice to see the result.
<img width="1331" alt="grafik" src="https://github.com/SoftFever/OrcaSlicer/assets/56330568/0dc26d46-6cef-41dd-a35c-ed871ec0ca4e">

**Reply by JeffDM2** [reply]
OK that makes some sense. But I don't understand why would they not have put the skirt outside the brim. The way you describe it seems like the arrangement is backwards. Even then it seems like an unnecessary complication of an extra feature, if I need more purge I could make the brim a little wider.

**Reply by Manfredo1952** [reply]
The brim is only needed with special filaments. If you print regular PLA or PETG, no need to worry about the brim. In that case a skirt with a distance of say 10 mm to the object might make sense. In some cases, a brim completely circlung around the object and attached to same can be difficult to detach.

**Comment by gircysm** [comment]
Revisiting this with V2.2.0-rc,
issue appears to be resolved by reducing `Elephant Foot Compensation`.
I suppose it makes some sense, but could also be argued to be a bug.
